#!/bin/sh /usr/share/dpatch/dpatch-run
## 03_manpage_fix.dpatch by Kartik Mistry <kartik.mistry@gmail.com>
##
## DP: Remove the useless paragraph from orca.1 manpage

@DPATCH@
--- gnome-orca-2.17.4.orig/docs/man/orca.1      2007-02-02 11:03:18.000000000 +0530
+++ gnome-orca-2.17.4/docs/man/orca.1   2007-02-02 11:03:52.000000000 +0530
@@ -102,14 +102,6 @@
 employ an assistive technology (AT) that exposes the graphical
 desktop information encoded in the GNOME Accessibility Framework\&.
 .PP
-Customers with blind employees/students need to have the ability
-to adapt and customize their assistive technology solutions - both 
-to improve the efficiency of their blind users (to help make up for
-disadvantages inherent in not being able to "see" the entire screen
-immediately at a glace), and also to "work around" poorly designed 
-and not particularly accessible applications that their blind users
-need to use\&.
-.PP
 orca is an extensible assistive technology that provides end-user
 access to applications and toolkits that support the GNOME 
 Accessibility Framework\&. It has been designed with direct continual

